vividus/src/main/java/org/vividus/configuration/ConfigurationResolver.java

```diff
@@ -99,6 +99,7 @@ public static ConfigurationResolver getInstance() throws IOException
 
         Properties overridingAndSystemProperties = new Properties();
         overridingAndSystemProperties.putAll(overridingProperties);
+        overridingAndSystemProperties.putAll(System.getenv());
```

Comment:
@valfirst  Why we don't have unit tests for the class?

Reply:
it contains a lot of logic which includes interaction with system resources and etc. the logic is covered by integration tests: https://github.com/vividus-framework/vividus/blob/master/vividus/src/integrationTest/java/org/vividus/configuration/BeanFactoryTests.java